src/Orleans/CodeGeneration/TypeUtils.cs

```diff
@@ -491,18 +491,6 @@ public static bool IsGrainMethodInvokerType(Type type)
             return generalType.IsAssignableFrom(type) && TypeHasAttribute(type, typeof(MethodInvokerAttribute));
         }
 
-#if NETSTANDARD_TODO
-        public static Type ResolveType(string fullName)
-        {
-            return Type.GetType(fullName, true);
-        }
-
-        public static bool TryResolveType(string fullName, out Type type)
-        {
-            type = Type.GetType(fullName, false);
-            return type != null;
-        }
-#else
```

Comment:
without this, the test cannot resolve type in another assembly. 

ASAIK, this two methods rely on CachedTypeResolver, which I also enabled in netstandard. I read through the class, I think the class itself doesn't have complication in netstandard. it's its dependency which relates to assembly loading which has complication. 

Maybe tell me more of what runtime behavior it will effect